tkg/tkgpackageclient/client.go

```diff
@@ -4,7 +4,7 @@
 package tkgpackageclient
 
 import (
-	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/kappclient"
+	"github.com/vmware-tanzu/tanzu-framework/tkg/kappclient"
```

Comment:
Is it possible that the subdirectories ```tkgpackageclient```, ```tkgpackagedatamodel```,```kappclient``` and ```tkg/test/tkgpackageclient``` remain where it is? I'm working to moving those to their own subdirectory and this change will cause big merge conflict